stoobly_agent/app/cli/helpers/handle_config_update_service.py

```diff
@@ -127,6 +127,21 @@ def handle_order_update(new_settings: Settings, context: Context = None):
         scenario_model = ScenarioModel(new_settings)
         scenario_model.update(_scenario_key.id, **{ 'overwritable': False })[1]
 
+def handle_strategy_update(new_settings: Settings, context: Context = None):
```

Comment:
The purpose of these function is to handle side effects related to config change. We should not be handling the config change itself here.